irohad/ametsuchi/impl/mutable_storage_impl.cpp

```diff
@@ -29,7 +29,11 @@ namespace iroha {
           command_executor_(std::move(cmd_executor)),
           committed(false),
           log_(logger::log("MutableStorage")) {
-      *sql_ << "BEGIN";
+      try {
+        *sql_ << "BEGIN";
+      } catch (std::exception &e) {
+        log_->error("Mutable storage initialization has been failed");
```

Comment:
```suggestion
         log_->error("Mutable storage initialization has failed");
```

Reply:
Also catching this exception seems bad, since now object is successfully created, however it cannot be used. Please consider removing try-catch here.